.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/sid/code/eslint-plugin-valtio/src/StateSnapshot.js":"1","/Users/sid/code/eslint-plugin-valtio/tests/StateSnapshot.test.js":"2","/Users/sid/code/eslint-plugin-valtio/src/lib/utils.js":"3"},{"size":11561,"mtime":1658412723010,"results":"4","hashOfConfig":"5"},{"size":8492,"mtime":1658412710156,"results":"6","hashOfConfig":"5"},{"size":8621,"mtime":1658412710156,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":1,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"wgawvu",{"filePath":"10","messages":"11","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"12","messages":"13","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/sid/code/eslint-plugin-valtio/src/StateSnapshot.js",["14"],"/Users/sid/code/eslint-plugin-valtio/tests/StateSnapshot.test.js",[],"/Users/sid/code/eslint-plugin-valtio/src/lib/utils.js",[],{"ruleId":"15","severity":2,"message":"16","line":399,"column":10,"nodeType":"17","messageId":"18","endLine":399,"endColumn":25},"no-unused-vars","'isInDeclaration' is defined but never used.","Identifier","unusedVar"]
```

Comment:
Don't you want to commit this file?

Reply:
The think is that while I've implemented the `use` check for custom hooks, it's right now a very tight implementation. 
I'd like it to be able to handle a few more cases and I haven't added those test cases in the commit since I'm still fixing the rules for it. 

The commit only has changes specific for the Issue it's linked to. 
Should be able to get the other cases working by evening today. 

There's just 2-3 more cases

Reply:
My question was if adding `.eslintcache` to the repo was intentional.

Reply:
Nah, it wasn't